ux.symfony.com/src/Twig/DemosList.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Twig;
+
+use App\Service\LiveDemoRepository;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+
+#[AsTwigComponent]
+readonly class DemosList
```

Comment:
What is the goal of this component (i mean, compared to the controller) ? 

We now create an isolated scope that we don't really need, no ?

Reply:
I made it to be able to use the template to generate the card for the cookbook